.travis.yml

```diff
@@ -22,7 +21,7 @@ matrix:
         - TOXENV=py35
 
 install:
-  - pip install -U tox
+  - pip install tox
```

Comment:
I'd strongly prefer we kept using `-U`; Travis has a bunch of Python modules installed out of the box, some of which are absolutely ancient (i.e., several years old). Given I've been bitten by things not updating them before and ending up with dependencies not being met before (le sigh pip and and its lack of any real dependency resolution) and things messily breaking, I'd much rather just guarantee we were using the latest version of tox and all its dependencies. Or just hardcode specific versions of all.